mlkem/poly.c

```diff
@@ -228,6 +228,14 @@ void poly_sub(poly *r, const poly *b)
   }
 }
 
+/* Include zeta table unless NTT, invNTT and mulcache computation
+ * have been replaced by native implementations. */
+#if !defined(MLK_USE_NATIVE_POLY_MULCACHE_COMPUTE) && \
+    !defined(MLK_USE_NATIVE_NTT) && !defined(MLK_USE_NATIVE_INTT)
+#include "zetas.inc"
```

Comment:
This is wrong isn't it? This will include it only if _all_ three are not defined, but you want to include it if _any_ of the three is not defined.

Reply:
Good spot, fixed